app/services/users.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const bcrypt = require('bcrypt'),
+  orm = require('./../orm'),
+  errors = require('../errors');
+
+exports.create = (user) => {
+  const saltRounds = 10;
```

Comment:
If you use always the same amount you could move this declaration outside the method.